lib/iris/tests/unit/fileformats/pp_rules/test__reduced_points_and_bounds.py

```diff
@@ -1,4 +1,4 @@
-# (C) British Crown Copyright 2014 - 2015, Met Office
+# (C) British Crown Copyright 2014 - 2016, Met Office
```

Comment:
Thanks for the spot.
I think it was changed and then changed back, so the header update is the only thing left.
In which case (I'll check), that will need removing when I squish.